proto/cosmos/tx/v1beta1/service.proto

```diff
@@ -95,9 +95,9 @@ message BroadcastTxRequest {
 enum BroadcastMode {
   // zero-value for mode ordering
   BROADCAST_MODE_UNSPECIFIED = 0;
-  // BROADCAST_MODE_BLOCK defines a tx broadcasting mode where the client waits for
-  // the tx to be committed in a block.
-  BROADCAST_MODE_BLOCK = 1;
+  // DEPRECATED: use BROADCAST_MODE_SYNC instead,
+  // BROADCAST_MODE_BLOCK is not supported by the SDK from v0.47.x onwards.
+  BROADCAST_MODE_BLOCK = 1 [deprecated = true];
```

Comment:
Curious what does `[deprecated = true];` do?

Reply:
https://developers.google.com/protocol-buffers/docs/proto3

```
deprecated (field option): If set to true, indicates that the field is deprecated and should not be used by new code. In most languages this has no actual effect. In Java, this becomes a @Deprecated annotation. In the future, other language-specific code generators may generate deprecation annotations on the field's accessors, which will in turn cause a warning to be emitted when compiling code which attempts to use the field. If the field is not used by anyone and you want to prevent new users from using it, consider replacing the field declaration with a reserved statement.
```